io/src/main/scala/aqua/files/Imports.scala

```diff
@@ -0,0 +1,56 @@
+package aqua.files
+
+import fs2.io.file.Path
+import scala.util.Try
+
+/**
+ * Imports resolution configuration.
+ */
+final case class Imports(
```

Comment:
there is two same classes with the same structure. Maybe merge it somehow?

Reply:
There are two of them so that `api` and `io` packages have clear dependencies. 
If we put `Imports` to `api`, `io` would have to depend on `api` and this would create cyclic dependency. Also `Imports` define the logic of resolving imports and `api` does not feel like the proper place for it.
If we put `Imports` to `io`, then `api` would have to expose internal `io` type in interface. This does not feel right to. 
So those two classes exist to decouple interface representation and representation for internal logic. If one of them changes, we would need to change only the conversion function. But now they are the same, yes.